airship_tempest_plugin/hacking/checks.py

```diff
@@ -144,7 +144,7 @@ def no_rbac_rule_validation_decorator(physical_line, filename):
     """
     global have_rbac_decorator
 
-    if ("airship_tempest_plugin/tests/api/rbac" in filename or
+    if ("airship_tempest_plugin/tests/api/shipyard/rbac" in filename or
```

Comment:
it shouldn't be just for shipyard as more rbac tests for other services could be added later

Reply:
This is added to bypass the naming convention validation for function tests; Naming convention validation would be applicable for all rbac tests only.

Reply:
ok, but maybe use regex so it is not hardcoded to only shipyard.

Reply:
this still needs to be updated. nothing here should be hardcoded to shipyard